.github/workflows/deploy_miljo.yml

```diff
@@ -9,8 +9,8 @@ jobs:
     runs-on: ubuntu-latest
     if: github.event.action == 'deploy_miljo_tag'
     steps:
-      - name: Will be deployed to q1?
-        if: github.event.client_payload.MILJO == 'q1'
+      - name: Will be deployed to dev-sbs?
```

Comment:
Du kan fjerne hele dette steget egentlig
` name: Will be deployed to dev-sbs? `

Reply:
Can do. Vet ikke bakgrunnen til at vi opprinnelig hadde dette steget 🤷

Reply:
Jeg vet. Det var jeg som testet noe. Men så ble jeg forstyrra av noe annet og kom aldri tilbake 😅 
GA har blitt så bra så vi får all info vi trenger derfra nå :)